lib/ASTGen/Sources/ASTGen/SourceFile.swift

```diff
@@ -122,7 +122,8 @@ public func parseSourceFile(
       .preambleMacroExpansion,
       .replacedFunctionBody,
       .prettyPrinted,
-      .defaultArgument:
+      .defaultArgument,
+      .attribute:
```

Comment:
Could you move `.attribute` to `. memberAttributeMacroExpansion` branch below, it's still not implemented and doing the same thing, but just not to forget to move it later.

Reply:
Okay, done!